src/iterator/_deprecated/group-by/group-by.ts

```diff
@@ -29,6 +29,7 @@ import { foldl } from '../fold/index.js'
  * @typeParam K - The key type.
  *
  * @group Iterators
+ * @deprecated Use `xs.groupBy(group)` instead
```

Comment:
this does not exist, does it? haven't found something like this in arrays/iterators

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/groupBy